src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -133,22 +163,38 @@ function PIASubHeader({
       primaryButtonText === SubmitButtonTextEnum.DELEGATE_FINISH_REVIEW
     ) {
       setDisableSubmitButton(true);
+    } else if (
+      enableComplete === false &&
+      primaryButtonText === SubmitButtonTextEnum.COMPLETE_PIA
+    ) {
+      setDisableSubmitButton(true);
     } else {
       setDisableSubmitButton(false);
     }
-  }, [enableFinalReview, isValidationFailed, mode, primaryButtonText]);
+  }, [
+    enableComplete,
+    enableFinalReview,
+    isValidationFailed,
+    mode,
+    primaryButtonText,
+  ]);
 
   const showSaveAndEditButton = () => {
     // we may revisit this part later for standard PIA
     if (mode === 'view' && pia.status === PiaStatuses.CPO_REVIEW) return false;
-    else if (mode === 'view' && pia.status === PiaStatuses.FINAL_REVIEW)
+    else if (
+      mode === 'view' &&
+      (pia.status === PiaStatuses.FINAL_REVIEW ||
+        pia.status === PiaStatuses.COMPLETE)
+    )
       return false;
+
     return true;
   };
   const showSubmitButton = () => {
     const owner = getGUID() === pia.createdByGuid ? true : false;
-    // if the status is final review, hide submit button
-    if (pia.status === PiaStatuses.FINAL_REVIEW) return false;
+    // if the status is completed, hide submit button
+    if (pia.status === PiaStatuses.COMPLETE) return false;
```

Comment:
Can we add this logic to the state table. IN the complete staus have a parameter submit button and do the evaluation there?